### ISSUE TITLE
setImageList()  doesn't remove old list

### ISSUE THREAD

**Issue by KhaledAlramam** [denzcoskun/ImageSlideshow#6]
When i reuse the Slider object with setImageList, it doesn't remove olde data and add dots to the old ones

**Comment by denzcoskun** [comment]
Hi, I fixed this problem new version 0.0.4. Thank you for your feedback. 
Just update your version with:
```Kt
implementation 'com.github.denzcoskun:ImageSlideshow:0.0.4'
```

**Comment by KhaledAlramam** [comment]
> Hi, I fixed this problem new version 0.0.4. Thank you for your feedback.
> Just update your version with:
> 
> ```kotlin
> implementation 'com.github.denzcoskun:ImageSlideshow:0.0.4'
> ```

Thanks a lot
